nxdrive/gui/view.py

```diff
@@ -394,29 +396,42 @@ def data(self, index: QModelIndex, role: int = REMOTE_PATH) -> Any:
         if role == self.REMOTE_PATH:
             return str(row["remote_path"])
         elif role == self.STATUS:
-            return str(row["status"].name)
+            status = str(row["status"].name)
+            if status == "DONE":
+                status = "COMPLETED"
+            return status
         elif role == self.DESCRIPTION:
             description = str(row["description"])
             if not description:
                 description = f"Session {row['uid']}"
             return description
-        elif role == self.CREATED_AT:
+        elif role == self.CREATED_ON:
+            label = "STARTED"
+            args = []
             datetime = get_date_from_sqlite(str(row["created_at"]))
-            if not datetime:
-                return "?"
-            # As date_time is in UTC
-            offset = tzlocal().utcoffset(datetime)
-            if offset:
-                datetime += offset
-            return Translator.format_datetime(datetime)
-        elif role == self.COMPLETED_AT:
+            if datetime:
+                label += "_ON"
+                # As date_time is in UTC
+                offset = tzlocal().utcoffset(datetime)
+                if offset:
+                    datetime += offset
+                args.append(Translator.format_datetime(datetime))
+            return self.tr(label, args)
+        elif role == self.COMPLETED_ON:
+            label = "COMPLETED" if str(row["status"].name) == "DONE" else "CANCELLED"
+            args = []
             datetime = get_date_from_sqlite(str(row["completed_at"]))
-            if not datetime:
-                return "?"
-            offset = tzlocal().utcoffset(datetime)
-            if offset:
-                datetime += offset
-            return Translator.format_datetime(datetime)
+            if datetime:
+                label += "_ON"
+                offset = tzlocal().utcoffset(datetime)
+                if offset:
+                    datetime += offset
+                args.append(Translator.format_datetime(datetime))
+            return self.tr(label, args)
+        elif role == self.PROGRESS:
+            label = "SESSION_PROGRESS"
+            args = [f"{row['uploaded']}/{row['total']}"]
```

Comment:
And that's even better because now we can use number formatting :)
Look how `sizeof_fmt()` is used in that file to do the same here.
Maybe adding spaces around the slash would be a good thing too?

Reply:
OK I mixed my thoughts. You can skip that review.

Reply:
But still, you have to use a delimiter.